.browserslistrc

```diff
@@ -1,4 +1,4 @@
->1%,
-Chrome > 20
-last 4 versions,
-Firefox ESR
+last 1 chrome version,
+last 1 firefox version,
+last 1 safari version,
+last 1 edge version
```

Comment:
not sure this is correct, we must support older browser than this, for example firefox ESR and chrome going back 1 year at least (to support TV browser for example that lag behind a bit )

Reply:
I see. How about this then?
```
safari > 0 or firefox > 0 or edge > 0 and > 1%,
chrome > 0 and last 2 years,
firefox esr,
```

To explain the choices:
While working this out, I realized that the last safari version refers to the technology preview. So I thought it would probably make sense to adopt a usage-based approach to these lesser used browsers.

This currently evaluates to:
```
chrome 79
chrome 78
chrome 77
chrome 76
chrome 75
chrome 74
chrome 73
chrome 72
chrome 71
chrome 70
chrome 69
chrome 68
edge 18
firefox 71
firefox 68
safari 13
```